include/deal.II/base/data_out_base.h

```diff
@@ -1383,9 +1383,26 @@ namespace DataOutBase
       bool operator() (const Point<3> &point_one,
                        const Point<3> &point_two) const
       {
-        return (point_one(0) < point_two(0)
-                || (!(point_two(0) < point_one(0)) && (point_one(1) < point_two(1)
-                                                       || (!(point_two(1) < point_one(1)) && point_one(2) < point_two(2)))));
+        const double eps = 1e-10;
```

Comment:
probably would not matter, but one may use something like `100*std::numeric_limits<double>::epsilon`.